packages/components/src/components/tab/tab.stories.ts

```diff
@@ -33,129 +46,61 @@ export const Default = {
 };
 
 /**
- * Use the `active` attribute to toggle the active state. Styling to indicate the active state is applied in the `sd-tab-group` component.
+ * Use the `variant` attribute to display different styles.
```

Comment:
Use the ”variant” attribute to change the appearance.

Reply:
Default:  Can be used independently or within components, often for full-page content.
Container: Highlighted tab and content linked to a background container, typically for specific sections like sub-pages or teasers.